python/lang/security/audit/dangerous-system-call.py

```diff
@@ -11,6 +11,30 @@
 # ruleid:dangerous-system-call
 os.system(something())
 
+# ruleid:dangerous-system-call
+getattr(os, "system")(something())
+
+# ruleid:dangerous-system-call
+dynamic_system_by_static_os = getattr(os, "system")
+dynamic_system_by_static_os(something())
+
+# ruleid:dangerous-system-call
+__import__("os").system(something())
+
+# ruleid:dangerous-system-call
+getattr(__import__("os"), "system")(something())
+
+# ruleid:dangerous-system-call
+dynamic_os = __import__("os")
+dynamic_os.system(something())
+
+# ruleid:dangerous-system-call
+getattr(dynamic_os, "system")(something())
```

Comment:
the develop test isn't passing because its matching starting from line 28 -- to fix this, you can add `dynamic_os = __import__("os")` to the line above line 32.

Reply:
Done